README.md

```diff
@@ -54,6 +54,12 @@ This library is distributed on `npm`. In order to add it as a dependency, run th
 $ npm install googleapis
 ```
 
+Each API is also released as a submodule. In order to add it as a dependency, run the following command, replacing `API_NAME` with the desired api:
```

Comment:
I have a few thoughts/questions here:
- We need to pick an option #1, and an option #2.  My initial instinct is to say "you should install the smaller split modules unless you have a reason to choose the big one".  If we agree that's what we want, I'd consider leading with split modules. 
- I would replace `<API_NAME>` with an actual API name.  People are going to blindly copy and paste that thing. 
- Should we consider adding a table here, similar to what we do with google-cloud-node, which lists the available modules?

Reply:
> * We need to pick an option #1, and an option #2.  My initial instinct is to say "you should install the smaller split modules unless you have a reason to choose the big one".  If we agree that's what we want, I'd consider leading with split modules.

I think this is fine, but we should probably lead with the root, otherwise we'll need to change a whole lot of documentation pointing to installing `googleapis`, and also because that method is tried and true. Thoughts?

> * Should we consider adding a table here, similar to what we do with google-cloud-node, which lists the available modules?
I was looking into it until I realized that the README is not auto-generated, so we'd need to convert it to a template, add a generation script, etc.... If you think it's worth it, LMK!

Reply:
@sofisl @JustinBeckwith there are 100s of APIs is my concern, has the potential to bloat the README.

What if we link to this search query instead: `https://www.npmjs.com/search?q=scope%3Agoogleapis`.